warehouse/models/intermediate/gtfs/gtfs_quality/int_gtfs_quality__naive_organization_service_dataset_full_join.sql

```diff
@@ -33,12 +33,17 @@ service_data AS (
         ON CAST(date_spine.date AS TIMESTAMP) BETWEEN dim._valid_from AND dim._valid_to
 ),
 
+-- we need to back-date regional feed type later, so will reference the dimension table directly
```

Comment:
nit: can this comment go on the new join to indicate it's for the backdating?

Reply:
removed this